custom/icds_reports/templates/icds_reports/dashboard.html

```diff
@@ -146,7 +146,7 @@ <h2 class="dashboard-title">
                                       <ul class="nav navbar-nav" role="menu">
                                           <li class="dropdown">
                                               <span class="pointer">
-                                                  <a href="{% static 'media/Dashboard_Training_Manual.pdf' %}" download>
+                                                  <a href="{% if have_access_to_dashboard_manual %}{% url 'download_manual'  domain %}{%else%}{%static 'media/Dashboard_Training_Manual.pdf' %}{% endif %}" download>
```

Comment:
Is this line intended to gate the release of this until we upload a training manual? I think it may be simpler to always redirect to `download_manual` and in that view (in rough pseudo-code):

```
try:
    manual = IcdsFile.get('manual')
except NotFound:
    manual = manual_from_static_file
return manual
```